packages/babel-plugin-lingui-macro/test/js-t.test.ts

```diff
@@ -1,5 +1,7 @@
 import { macroTester } from "./macroTester"
 
+describe.skip("", () => {})
+
```

Comment:
this added intentionally to help IDE to understand that this file is executable jest test, otherwise it doesnt have an option to run tests in this file:

<img width="404" alt="image" src="https://github.com/lingui/js-lingui/assets/1586852/c9c93f2d-4931-47b1-9354-5e3e11386a23">